src/main/java/mate/academy/springboot/datajpa/service/mapper/ResponseDtoMapper.java

```diff
@@ -0,0 +1,5 @@
+package mate.academy.springboot.datajpa.service.mapper;
+
+public interface ResponseDtoMapper<M, D> {
+    D modelToDto(M model);
```

Comment:
```suggestion
    D mapToDto(M model);
```
I think it should be similar naming of both mapping methods.